magento/adobe-stock-integration
# [Refactoring] Refactor js Components && SQL Optimization

### Description (*)


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1610:  [Refactoring] SQL optimization
2. Fixes magento/adobe-stock-integration#1613: [Refactoring] JS
3. Fixes magento/adobe-stock-integration#1631: [Refactoring] Remove the category mention from the image-details.jso 

### Manual testing scenarios (*)
1. Need to verify all cases with MediaGallery filters
2. Need to verify all behavior with delete images, upload images, verify count where images is used in
3. License images from media gallery
4. Verify information in image details edit image details
5. Create directory delete directory, also upload images in batches
6. Insert images to wysywig
7. Mass action mode for media gallery
